charge.go

```diff
@@ -344,6 +345,12 @@ type ChargePaymentMethodDetailsEps struct {
 	VerifiedName string `json:"verified_name"`
 }
 
+// ChargePaymentMethodDetailsFpx represents details about the FPX PaymentMethod.
+type ChargePaymentMethodDetailsFpx struct {
+	Bank          string `json:"bank"`
```

Comment:
You're missing `account_holder_type` and it should be a list of constants

Reply:
@remi-stripe  done fixing. thanks for the review.